docs/components/messageListener.md

```diff
@@ -0,0 +1,168 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.messageListener`
+
+This component listens to the messages and signals sent by the [`inputMapper`](inputMapper.md) component in the content
+scripts and then calls the action invokers according to the received message.
+
+## Using this grade
+
+The component can be used by creating its instance or by making a custom component using the grade.
+
+``` javascript
+// Either create an instance of the messageListener.
+var messageListenerInstanceOne = gamepad.messageListener();
+
+// Otherwise create a custom component using the messageListener grade.
+fluid.defaults("my.messageListener.grade", {
+    gradeNames: ["gamepad.messageListener", "my.CustomGrade"]
+});
+var messageListenerInstanceTwo = my.messageListener.grade();
+```
+
+## Component Options
+
+This component does not provide any configuration options.
+
+## Non-navigation Invokers
+
+### `{messageListener}.addListener()`
+
+- Returns: Nothing.
+
+Listens to the messages received from the [`inputMapper`](inputMapper.md) component in the content scripts and triggers
+the [actionExecutor](#messagelisteneractionexecutoractiondata) invoker as a callback function.
```

Comment:
I had to refresh my own memory, `addListener` is a method provided by individual events.  I do think it would be better to find a name that is not already widely used in other parts of Infusion.

Reply:
From this comment and the previous one, I do think it'd be better if we use some other name for the invoker. Maybe should I add a TODO for now?